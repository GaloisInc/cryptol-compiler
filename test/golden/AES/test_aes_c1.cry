import AES
main = aesEncrypt (0x00112233445566778899aabbccddeeff, 0x000102030405060708090a0b0c0d0e0f)