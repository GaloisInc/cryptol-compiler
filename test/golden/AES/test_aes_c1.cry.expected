0x69c4e0d86a7b0430d8cdb78070b4c55a
